CONTRIBUTION-zh.md

```diff
@@ -0,0 +1,80 @@
+# Contribution (Draft)
+
+Awesome Satori 是用于展示 Satori protocol 社区中各种有趣主题的资源列表，旨在收集和分享与 Satori 相关优秀资源，包括协议实现、SDK、Framework、教程、工具等，帮助其他人学习、发现和使用 Satori。
+为了保证质量和一致性，本篇文档将详细描述您应该如何贡献作品。
+
+[English](./CONTRIBUTION.md) | 简体中文
+
+## 准备工作
+
+在贡献您的作品前，我们需要一点准备工作：
+
+- 拥有一个 GitHub 账号，并 Fork 本仓库
```

Comment:
Every lines in the list are "sentences", so you need to add a period at the end.
Also English one.